README.md

```diff
@@ -1,73 +1,52 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Africa Green Energy Summit 2022
 
-> Description the project.
+> This is a dising for a Developers summit.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- javascript
+- Git & Github
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://github.com/Duncannjau/Africa-Green-Energy-Summit
+)
```

Comment:
- [ ] Please deploy your app correctly using GitHub pages by selecting the correct branch that your work is on. This link only leads to your repository.

- [ ] Please you are required to record a video presenting the features of the project. Add the link to the video in your PR description and README.md file.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, showing the pages in different screen sizes.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.